<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scene_widget</class>
 <widget class="QWidget" name="scene_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tavern</string>
  </property>
  <widget class="QTabWidget" name="preset_tabs">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>-2</y>
     <width>791</width>
     <height>184</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ActionsContextMenu</enum>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="brawl_tab">
    <attribute name="title">
     <string>brawl</string>
    </attribute>
    <widget class="QScrollArea" name="channel_scroll_area_3">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>789</width>
       <height>152</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>787</width>
        <height>150</height>
       </rect>
      </property>
      <widget class="QScrollBar" name="horizontalScrollBar_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>132</y>
         <width>791</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="festive_tab">
    <attribute name="title">
     <string>festive</string>
    </attribute>
    <widget class="QScrollArea" name="channel_scroll_area_2">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>789</width>
       <height>152</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>787</width>
        <height>150</height>
       </rect>
      </property>
      <widget class="QScrollBar" name="horizontalScrollBar_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>132</y>
         <width>791</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="morning_tab">
    <attribute name="title">
     <string>morning</string>
    </attribute>
    <widget class="QScrollArea" name="channel_scroll_area">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>789</width>
       <height>152</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>787</width>
        <height>150</height>
       </rect>
      </property>
      <widget class="QScrollBar" name="horizontalScrollBar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>132</y>
         <width>791</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="scene_frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>111</width>
     <height>181</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="mute_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>40</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/icons8-no-audio-30.png</normaloff>icons/icons8-no-audio-30.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="balance_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>93</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>L               R</string>
    </property>
   </widget>
   <widget class="QDial" name="balance_pot">
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>41</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="scene_label">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>-1</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cantarell Extra Bold</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Tavern: Preset</string>
    </property>
   </widget>
   <widget class="QSlider" name="volume_slide">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>33</y>
      <width>21</width>
      <height>71</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="horizontal_line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>18</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="vertical_line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>27</y>
      <width>20</width>
      <height>104</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="preset_list">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>brawl</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>morning</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>night</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>festive</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="play_next_button">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/icons8-end-30.png</normaloff>icons/icons8-end-30.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="proxy_button">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>110</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/icons8-proximity-sensor-30.png</normaloff>icons/icons8-proximity-sensor-30.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="timed_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/icons8-sand-timer-30.png</normaloff>icons/icons8-sand-timer-30.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="settings_button">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/icons8-settings-30.png</normaloff>icons/icons8-settings-30.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="play_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/icons8-play-30.png</normaloff>icons/icons8-play-30.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="expand_button">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/icons8-adjustment-100.png</normaloff>icons/icons8-adjustment-100.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <zorder>mute_button</zorder>
   <zorder>balance_label</zorder>
   <zorder>balance_pot</zorder>
   <zorder>scene_label</zorder>
   <zorder>volume_slide</zorder>
   <zorder>horizontal_line</zorder>
   <zorder>preset_list</zorder>
   <zorder>vertical_line</zorder>
   <zorder>play_next_button</zorder>
   <zorder>proxy_button</zorder>
   <zorder>timed_button</zorder>
   <zorder>settings_button</zorder>
   <zorder>play_button</zorder>
   <zorder>expand_button</zorder>
  </widget>
  <action name="actionNew_Preset">
   <property name="text">
    <string>New Preset</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
